From eb2820d0438731f7bf948e33f7e68ae557a6a72c Mon Sep 17 00:00:00 2001
From: Selvin Xavier <selvin.xavier@emulex.com>
Date: Thu, 19 Mar 2015 11:56:10 +0530
Subject: [PATCH 3/7] be2net: Fixing the issue in enabling MSIX in RHEL 6.6

HAVE_PCI_ENABLE_MSIX_RANGE define was not used properly while
enabling MSIX

Signed-off-by: Selvin Xavier <selvin.xavier@emulex.com>
---
 drivers/net/ethernet/emulex/benet/be_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index c745fba..2492c7c 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -2903,6 +2903,7 @@ static int be_msix_enable(struct be_adapter *adapter)
 	 }
 
 done:
+#endif
 	if (be_roce_supported(adapter) && num_vec > MIN_MSIX_VECTORS) {
 		adapter->num_msix_roce_vec = num_vec / 2;
 		dev_info(dev, "enabled %d MSI-x vector(s) for RoCE\n",
@@ -2914,7 +2915,6 @@ done:
 	dev_info(dev, "enabled %d MSI-x vector(s) for NIC\n",
 		 adapter->num_msix_vec);
 	return 0;
-#endif
 
 fail:
 	dev_warn(dev, "MSIx enable failed\n");
-- 
1.7.9.5

